source/warble/main.py

```diff
@@ -0,0 +1,14 @@
+import sys
```

Comment:
This file should be named something like graalvisor-main.py or graalvisor-entrypoint.py.

Reply:
BTW, we have been spelling Graalvisor and not GraalVisor.